### PR TITLE
Added a new controller (about) and copyright page

### PR DESCRIPTION
**Pull request by gortonz**
A pull request for adding copyright page

### CODE REVIEW

**Comment by catherine-jones** [comment]
hey @gortonz about.coffee, about.scss and about.html.erb can be deleted as we wont use these files and you can probably remove the about helper as well. Other wise LGTM and you can merge once changed are complete

File: app/views/about/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>About#about</h1>
```
Comment (elisechant) [inline app/views/about/about.html.erb]: why "About#about" ??

File: app/views/about/copyright.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="container">
+  <h1>Copyright</h1>
+  <p>With the exception of the Commonwealth Coat of Arms and where otherwise noted, all material presented on this site
+    is provided under the latest Creative Commons Attribution licence.</p>
+  <p>The details of the <a href="http://creativecommons.org/licenses/by/4.0/">relevant licence conditions</a> and the
+    <a
+    href="http://creativecommons.org/licenses/by/4.0/legalcode">full legal code</a> for the licence are available on the Creative Commons website.</p>
+  <h2>Use of the Coat of Arms</h2>
+  <p>The terms under which the Coat of Arms can be used are detailed on the <a href="http://www.itsanhonour.gov.au/coat-arms/index.cfm">It’s an Honour</a>        website.</p>
```
Comment (elisechant) [inline app/views/about/copyright.html.erb]: fix whitespace before "website"

**Comment by elisechant** [comment]
Question? why do have an "about" page?

File: app/decorators/about_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class AboutDecorator < Draper::Decorator
```
Comment (tobyhede) [inline app/decorators/about_decorator.rb]: This can be deleted too.

File: app/controllers/about_controller.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class AboutController < ApplicationController
+  def copyright
+  end
+
+  def about
```
Comment (tobyhede) [inline app/controllers/about_controller.rb]: Let's make this the `index` method of the About controller. Then the path will be about#index.

File: app/views/about/about.html.erb (hunk 1 above)
Comment (gortonz) [inline app/views/about/about.html.erb]: I thought we would want a about page. I will delete it if we do not need one

**Comment by catherine-jones** [comment]
@elisechant RE: why do we have an about page i think @gortonz used one of the rails helpers to generate all of the pages.

**Comment by catherine-jones** [comment]
@gortonz LGTM

**Comment by gortonz** [comment]
I deleted the about page and added a few minor fixes.

**Comment by catherine-jones** [comment]
@gortonz yep feel free to merge when ready

**Pull request merged**